web/react/components/file_preview.jsx

```diff
@@ -27,14 +28,14 @@ module.exports = React.createClass({
 
             if (type === "image") {
                 previews.push(
-                    <div key={filename} className="preview-div" data-filename={filename}>
+                    <div key={filename} className="preview-div" data-filename={originalFilename}>
```

Comment:
Because the parent function (removePreview) uses the data-filename attribute to remove the file previews and it only has the original filename before manipulation. We could re-manipulate it in removePreview() but that's more work and doesn't really make sense if we can just use the original name here.